src/main/java/com/cityelf/repository/NotificationRepository.java

```diff
@@ -15,7 +15,7 @@
   private UserRepository userRepository;
 
   public Optional<Notification> getNotification(long id) {
-    Optional<User> innerUser = Optional.ofNullable(userRepository.getUser(id));
+    Optional<User> innerUser = userRepository.getUser(id);
```

Comment:
Combine this 2 lines together
return userRepository.getUser(id)
    .map(...);